src/Azure.Functions.Cli/Common/Utilities.cs

```diff
@@ -171,5 +179,76 @@ internal static string EnsureCoreToolsLocalData()
             FileSystemHelpers.EnsureDirectory(localPath);
             return localPath;
         }
+
+        internal static LogLevel GetHostJsonDefaultLogLevel(IConfigurationRoot hostJsonConfig)
+        {
+            string defaultLogLevelKey = $"AzureFunctionsJobHost:logging:loglevel:default";
```

Comment:
Two nits:
1. Use `ConfigurationSectionNames.JobHost`
2. You can build these using `ConfigurationPath.Combine()`... a lot like `Path.Combine()`